Tests/SwiftStdlibTests/RangeReplaceableCollectionTests.swift

```diff
@@ -11,6 +11,13 @@ import XCTest
 
 class RangeReplaceableCollectionTests: XCTestCase {
 
+    func testInitExpressionOfSize() {
```

Comment:
I'm missing the test for condition the size negative or 0 within the test. Can we add that?